packages/mixer/src/components/Withdraw/Withdraw.tsx

```diff
@@ -129,7 +131,12 @@ export const Withdraw: React.FC<WithdrawProps> = () => {
         }
       });
   };
-
+  const relayers = relayersState.relayers;
+  const activeRelayer = relayersState.activeRelayer;
+  console.log({
+    relayersState,
+    relayers,
+  });
```

Comment:
Can we do some logging cleanup / leave ones in that are helpful?

Reply:
removed, will add a cleanup task for console logs from the whole dApp with #243